### PR TITLE
chore: zksync Airdrop banner and modal

### PR DESCRIPTION
**Pull request by Chef-Yogi**
---

## PR-Codex overview
The focus of this PR is to implement the ZkSync token airdrop feature, including contracts, banners, and gas sponsorship balance.

### Detailed summary
- Added ZkSync token airdrop contract and address helpers
- Implemented ZkSync airdrop banner
- Created gas sponsorship balance hook
- Updated translations for ZkSync airdrop
- Whitelisted contracts for paymaster interaction

> The following files were skipped due to too many changes: `apps/web/src/views/Home/components/Banners/ZksyncAirdrpoBanner.tsx`, `apps/web/src/components/ClaimZksyncAirdropModal/hooks.tsx`, `apps/web/src/components/ClaimZksyncAirdropModal/index.tsx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/web/src/config/abi/zksynAirdrop.ts
Comment (chefjackson) [inline apps/web/src/config/abi/zksynAirdrop.ts:1]: let's minimize the abi if possible and only keep those used on FE

File: apps/web/src/config/paymaster.ts [hunk 1]
```diff
@@ -93,6 +96,17 @@ export const paymasterInfo: {
...
 
+/**
+ * Contracts that the paymaster is allowed to interact with
+ */
+export const PAYMASTER_CONTRACT_WHITELIST = [
+  getUniversalRouterAddress(ChainId.ZKSYNC), // Universal Router on zkSync
+  getAddressFromMap(addresses.zkSyncAirDrop, ChainId.ZKSYNC), // ZKSync AirDrop
+
+  // Token addresses for Approval
+  ...paymasterTokens.map((token) => token.wrapped.address),
```
Comment (chefjackson) [inline apps/web/src/config/paymaster.ts:107]: Is it gonna block the approval of tokens outside this list?

Comment (thechefpenguin) [reply apps/web/src/config/paymaster.ts:107]: resolving this

File: apps/web/src/config/abi/zksynAirdrop.ts
Comment (Chef-Yogi) [reply apps/web/src/config/abi/zksynAirdrop.ts:1]: Done

File: apps/web/src/views/Home/components/Banners/ZksyncAirdrpoBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  return (
+    <>
+      <BannerContainer background="linear-gradient(180deg, rgba(39, 78, 234, 0.88) 74.72%, rgba(48, 90, 235, 0.88) 85.09%, rgba(74, 127, 239, 0.88) 94.31%, rgba(149, 231, 249, 0.88) 100%), linear-gradient(180deg, rgba(32, 68, 197, 0.20) 0%, rgba(65, 57, 168, 0.00) 100%), linear-gradient(0deg, #264FED 0%, #264FED 100%), linear-gradient(180deg, #CCA382 0%, #9DC38F 49.77%, #9FCCCF 100%);">
+        <BannerMain
+          badges={
+            <StyledFlexContainer gap="8px">
+              <PancakeSwapBadge whiteText />
+              {!(isMobile || isMd) ? (
+                <Text bold color="white">
+                  Perpetual v2
```
Comment (thechefpenguin) [inline apps/web/src/views/Home/components/Banners/ZksyncAirdrpoBanner.tsx:85]: Need to remove Perpetual badge and fix width on ZKSync logo

Comment (Chef-Yogi) [reply apps/web/src/views/Home/components/Banners/ZksyncAirdrpoBanner.tsx]: fixed

**Review by chefjackson — APPROVED** [review]

**Review by thechefpenguin — APPROVED** [review]

**Pull request merged**